gen/tests/test_adminrouter_tls_conf.py

```diff
@@ -57,7 +169,7 @@ class TestToggleTLS1:
     def supported_ssl_protocols(self, new_config_arguments) -> List[str]:
```

Comment:
Lets rename this function to include `master` in name to indicate that it checks for protocol version support in AR master configuration.